examples/py/example.py

```diff
@@ -50,9 +50,9 @@ def get_closest(x, y, units):
     cl = tc.Client()
     cl.connect(args.hostname, args.port)
     state = cl.init(micro_battles=True)
-    for plid, race in enumerate(state.player_races):
-        print("player {} named {} is {}".format(plid, state.player_names[plid],
-            tc.Constants.races._dict[race]))
+    for pid, player in state.player_info.items():
+        print("player {} named {} is {}".format(player.id, player.name,
```

Comment:
I'm assuming you tested this and it printed out the right things? Are neutral players handled correctly here?

Reply:
I tested all frontends by connecting to a game a couple of times. The neutral player is included in the map with race set to "Unused" and is_enemy set to false.